tsconfig.json

```diff
@@ -14,6 +14,8 @@
     "lib": [
       "es2016",
       "dom"
-    ]
+    ],
+    "module": "es2015",
+    "baseUrl": "./"
```

Comment:
One of the steps on the upgrade checklist was to tell the Angular compiler specifically to stop preserving whitespace: PRX/publish.prx.org@193d8a6dd80

It's not really made clear why, but I made the change and just assumed it was added to the list for a good reason.

Reply:
sure missed that, thanks!